src/xia2/Test/regression/test_multiplex.py

```diff
@@ -30,13 +32,21 @@
 ]
 
 
-def test_proteinase_k(mocker, regression_test, dials_data, tmpdir):
+@pytest.fixture()
+def proteinase_k(regression_test, dials_data, tmp_path):
     data_dir = dials_data("multi_crystal_proteinase_k")
     expts = sorted(f.strpath for f in data_dir.listdir("experiments*.json"))
     refls = sorted(f.strpath for f in data_dir.listdir("reflections*.pickle"))
+    os.chdir(tmp_path)
```

Comment:
this looks like asking for trouble

Reply:
Hmm, I don't like it took much, it is a bit unexpected/implicity. OTOH, it does remove five `tmpdir.as_cwd()` and their respective indent blocks.

I wonder if putting the tmpdir context manager around the yield would be better?